apis/python/src/tiledbsoma/tiledb_object.py

```diff
@@ -67,6 +67,10 @@ def _default_ctx(self) -> tiledb.Ctx:
 
         return tiledb.Ctx(cfg)
 
+    def metadata(self) -> MetadataMapping:
```

Comment:
`metadata` is defined in `somacore` as a `@property`, not a function.

Reply:
@thetorpedodog I cannot explain then why I had to change _from property to function_ to make errors go away.

Maybe not everything is at `HEAD` in both repos -- ???

Reply:
@thetorpedodog addressed on commit
https://github.com/single-cell-data/TileDB-SOMA/pull/673/commits

Reply:
I think this *might* be the issue: if an abstract `@property` is defined, it needs to be defined as a property on the implementing class:

```python
class Base(metaclass=abc.ABCMeta):
  @property
  @abc.abstractmethod
  def it(self): pass

class ThisWillWork(Base):
  @property
  def it(self): return 50

class ThisFails(Base):
  def __init__(self):
    self.it = 50
```